backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3263,4 +3263,60 @@ databaseChangeLog:
         - sql:
             remarks: Populate new `emails` column with value from patient `email`
             sql: |
-              UPDATE person SET emails = ARRAY[email] WHERE email IS NOT NULL;
\ No newline at end of file
+              UPDATE person SET emails = ARRAY[email] WHERE email IS NOT NULL;
+  - changeSet:
+      id: add-specimen_type-to-test_event
```

Comment:
I commented on some inconsistencies in these `id` fields in a previous PR. It looks like the name of the column is missing its `_id` suffix here and on line 3287. Not a dealbreaker, but I would love to see this be consistent across all three of your `changeSet` objects.

Reply:
changed

Reply:
Thank you! 😄